### PR TITLE
Moves mailchimp vars to env

### PR DESCRIPTION
**Pull request by TheDudeWithTheThing**
Left interest ids even though they're not used, I also need the right interest ids for test, which I'll modify in the other PR.

### CODE REVIEW

**Comment by markottaviani** [comment]
For the test mailchimp account (pulled from the mailchimp API Playground)

```
interest_supporter: = 'ce0f3791c7'
interest_active_application: = '5946f7f039'
interest_adopted_from_oph: = '003e2d991a'
```

**Comment by TheDudeWithTheThing** [comment]
@markottaviani all updated, make sure you add the env vars before deploying (duh)

File: db/seeds.rb [hunk 1]
```diff
@@ -14,6 +14,7 @@
           email: "test@test.com",
           password: "foobar99",
           password_confirmation: "foobar99",
+          state: 'NY'
```
Comment (TheDudeWithTheThing) [inline db/seeds.rb]: I upgraded postgres and had to start over, this was missing from the seed file, totally unrelated to what's trying to be done here.

**Comment by markottaviani** [comment]
Looks good!  I'll get this tested and deployed tonight

**Comment by markottaviani** [comment]
testing done!

**Comment by TheDudeWithTheThing** [comment]
and it worked???

**Comment by markottaviani** [comment]
Yup! Everything is working so far!

**Pull request merged**